appserver/common/glassfish-ee-api/src/main/java/com/sun/appserv/ClassLoaderUtil.java

```diff
@@ -66,123 +67,69 @@
-     *@param innerClassName the fully-qualified name of the inner class of interest
-     */
-    private static Class getInnerClass(Class cls, String innerClassName) {
-        Class result = null;
-        Class[] innerClasses = cls.getDeclaredClasses();
-        for (Class c : innerClasses) {
-            if (c.getName().equals(innerClassName)) {
-                result = c;
-                break;
-            }
-        }
-        return result;
-    }
     
     /**
-     *Releases resources held by the URLClassLoader.  Notably, close the jars
-     *opened by the loader. This does not prevent the class loader from
-     *continuing to return classes it has already resolved.
-     *@param classLoader the instance of URLClassLoader (or a subclass) 
-     *@return array of IOExceptions reporting jars that failed to close
+     * Releases resources held by the URLClassLoader.  Notably, close the jars
+     * opened by the loader. This does not prevent the class loader from
+     * continuing to return classes it has already resolved.
+     *
+     * @param classLoader the instance of URLClassLoader (or a subclass) 
+     *
+     * @return array of IOExceptions reporting jars that failed to close
      */
     public static void releaseLoader(URLClassLoader classLoader) {
         releaseLoader(classLoader, null);
     }
     
     /**
-     *Releases resources held by the URLClassLoader.  Notably, close the jars
-     *opened by the loader. This does not prevent the class loader from
-     *continuing to return classes it has alrady resolved although that is not
-     *what we intend to happen.  Initializes and updates the Vector of 
-     *jars that have been successfully closed.
-     *<p>
-     *Any errors are logged.
-     *@param classLoader the instance of URLClassLoader (or a subclass) 
-     *@param jarsClosed a Vector of Strings that will contain the names of jars 
-     * successfully closed; can be null if the caller does not need the information returned
-     *@return array of IOExceptions reporting jars that failed to close; null
-     *indicates that an error other than an IOException occurred attempting to
-     *release the loader; empty indicates a successful release; non-empty 
-     *indicates at least one error attempting to close an open jar.
+     * Releases resources held by the URLClassLoader.  Notably, close the jars
+     * opened by the loader. This does not prevent the class loader from
+     * continuing to return classes it has already resolved although that is not
+     * what we intend to happen. Initializes and updates the Vector of 
+     * jars that have been successfully closed.
+     *
+     * <p>
+     * Any errors are logged.
+     *
+     * @param classLoader the instance of URLClassLoader (or a subclass) 
+     * @param jarsClosed a Vector of Strings that will contain the names of jars 
+     *        successfully closed; can be null if the caller does not need the information returned
+     * 
```

Comment:
Pretty sure our code style is no random empty lines between the tags in javadoc
(ﾉ≧∇≦)ﾉ ﾐ ┸━┸

Reply:
It's actually to separate the `param` ones from the `return` one ;)